jira/client.py

```diff
@@ -2927,6 +2927,19 @@ def statuses(self) -> List[Status]:
         ]
         return statuses
 
+    def projectstatuses(self, projectIdOrKey: str) -> List[Status]:
```

Comment:
(non-blocking)

@studioj what are you thoughts on using the more conventional naming convention here?

e.g.:

```suggestion
    def project_statuses(self, projectIdOrKey: str) -> List[Status]:
```

My only concern is that we don't really conform to this anyway, e.g. `statuscategories()` below. So perhaps we want to have a separate merge request where we add a deprecation warning to the current non snake_case methods, and also convert them to a wrapper of the snake case ones. Something like:

```python

def projectstatuses(self, projectIdOrKey: str) -> List[Status]:
    warnings.warn("Please use project_statuses()", DeprecationWarning)
    return self.project_status(projectIdOrKey)
   

def project_statuses(self, projectIdOrKey: str) -> List[Status]:
    ... 
```

Reply:
i would indeed do that in a separate PR, but tbh if we add something new i would suggest making it snake case yes

Reply:
so this method should be snake case
project_statuses